### DISCUSSION TITLE
Go-e Firmware 56.8 and API-v2 / Modbus

### DISCUSSION THREAD

**Discussion by rainer4** [goecharger/go-eCharger-API-v2#228] [General]
After installing Go-e Firmware 56.8 I have some Modbus issues:
e.g. register 100 (car state) was 1 "idle" in 56.2 when no car was connected, with 56.8 i receive 0 "unknown/defect"
other registers like 299 (Ampere_Volatile) worked
Are there any changes regarding Modbus registers with 56.8 Firmware?

**Comment by erxbout** [comment]
solved (#229)
